sentry-delayed_job/spec/spec_helper.rb

```diff
@@ -1,4 +1,5 @@
 require "bundler/setup"
+require "debug" if RUBY_VERSION.to_f >= 2.6 && RUBY_ENGINE == "ruby"
 require "pry"
```

Comment:
Not related: do you still use pry?

Reply:
nope! I've moved to `debugger` if available!